src/flow.h

```diff
@@ -330,6 +340,10 @@ typedef struct Flow_
     };
     uint8_t proto;
     uint8_t recursion_level;
+    uint8_t min_ttl_toserver;
```

Comment:
please move below vlan_id so all the fields used for hashing are together

Reply:
OK I do it, TTL variables are moved below vlan_id.